src/templates/common/phoneNumber/index.tsx

```diff
@@ -73,14 +74,15 @@ export const PhoneNumber = (
 
   return (
     <p className={className || undefined} data-testid="phone">
-      <strong>{labelToDisplay}: </strong>
+      <strong>{`${labelToDisplay}: `}</strong>
```

Comment:
Was this giving you some grief before? 🤔

Reply:
It was rendering with a line break in html but not on screen which is bad for screen readers and test assertions.
![Screenshot 2025-05-01 at 9 13 54 AM](https://github.com/user-attachments/assets/9f2e2e9b-0ef7-4f7c-a13d-44ff53530f28)